File: eval/fixtures/php-taskflow/src/Store/Priority.php
<?php

declare(strict_types=1);

namespace App\Store;

/** Task priority — a backed enum, used as a Task property type. */
enum Priority: string
{
    case Low = 'low';
    case Normal = 'normal';
    case High = 'high';

    public function weight(): int
    {
        return match ($this) {
            Priority::Low => 1,
            Priority::Normal => 2,
            Priority::High => 3,
        };
    }
}

File: eval/fixtures/python-taskflow/taskflow/store/priority.py
"""Task priority — an enum used as a Task attribute type."""

from enum import Enum


class Priority(Enum):
    LOW = "low"
    NORMAL = "normal"
    HIGH = "high"

    def weight(self) -> int:
        return {Priority.LOW: 1, Priority.NORMAL: 2, Priority.HIGH: 3}[self]
